### PR TITLE
Oceanwater 943 Update Stow and Unstow

### PR DESCRIPTION
**Pull request by kmdalal**
## Linked Issues:
| EPIC ⚡ | [OCEANWATER-933](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-933) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-943](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-943) |
| Github :octocat:  | #302 |

### Summary of Changes
* Remove goal, feedback, and result components of Stow and Unstow actions.
* Rename these actions to ArmStow and ArmUnstow.
* Move these action files to owl_msgs package.

### Test
* This branch must be tested with the corresponding branch in `ow_autonomy`.
* Do a clean build.  This itself verifies most of renaming and moving.
* Start any simulator world
* Run `arm_unstow_action_client.py` and see that the arm unstows normally.
* Run `arm_stow_action_client.py` and see that the arm stows normally.

### CODE REVIEW

**Comment by AstroStucky** [comment]
We should name the action client scripts `arm_unstow.py` and `arm_stow.py`.

**Comment by kmdalal** [comment]
> We should name the action client scripts `arm_unstow.py` and `arm_stow.py`.

That's right - done.

File: ow_lander/src/ow_lander/actions.py [hunk 1]
```diff
@@ -111,27 +111,27 @@ def ground_detect_cb():
...
-class UnstowServer(ArmTrajectoryMixin, ActionServerBase):
+class ArmUnstowServer(ArmTrajectoryMixin, ActionServerBase):
 
-  # UNIFICATION TODO: rename "Stow" to "ArmStow"
-  name          = 'Unstow'
-  action_type   = ow_lander.msg.UnstowAction
-  goal_type     = ow_lander.msg.UnstowGoal
-  feedback_type = ow_lander.msg.UnstowFeedback
-  result_type   = ow_lander.msg.UnstowResult
+  # UNIFICATION TODO: rename "ArmStow" to "ArmArmStow"
```
Comment (AstroStucky) [inline ow_lander/src/ow_lander/actions.py:116]: If unification turns `Stow` to `ArmStow` it follows that `ArmArmStow` is double-unification 🤣 
```suggestion
```
I kid. This comment is no long needed.

File: ow_lander/src/ow_lander/actions.py [hunk 2]
```diff
@@ -111,27 +111,27 @@ def ground_detect_cb():
...
-class StowServer(ArmTrajectoryMixin, ActionServerBase):
+class ArmStowServer(ArmTrajectoryMixin, ActionServerBase):
 
-  # UNIFICATION TODO: rename "Stow" to "ArmStow"
-  name          = 'Stow'
-  action_type   = ow_lander.msg.StowAction
-  goal_type     = ow_lander.msg.StowGoal
-  feedback_type = ow_lander.msg.StowFeedback
-  result_type   = ow_lander.msg.StowResult
+  # UNIFICATION TODO: rename "ArmStow" to "ArmArmStow"
```
Comment (AstroStucky) [inline ow_lander/src/ow_lander/actions.py:129]:
```suggestion
```

**Review by AstroStucky — APPROVED** [review]
Test works and code looks good. Just two comments lines that need to be removed.

File: ow_lander/src/ow_lander/actions.py (hunk 1 above)
Comment (kmdalal) [reply ow_lander/src/ow_lander/actions.py]: Search/replace error.  Both comments removed.

**Pull request merged**